src/main/java/uk/gov/hmcts/reform/jobscheduler/model/HttpAction.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.reform.jobscheduler.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import org.springframework.http.HttpMethod;
+
+import java.util.Map;
+
+public class HttpAction {
+
+    public final String url;
+    public final HttpMethod method;
+    public final Map<String, String> headers;
+    public final JsonNode body;
```

Comment:
Is it agreed that the body will always be JSON?

Reply:
Logic Apps expects json object https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-workflow-actions-triggers#http-action